docs/faq.md

```diff
@@ -415,7 +415,33 @@ jib.container.useCurrentTimestamp = true
 
 ### I would like to run my application with a javaagent.
 
-See [Can I ADD a custom directory to the image?](#can-i-add-a-custom-directory-to-the-image).
+See also [Can I ADD a custom directory to the image?](#can-i-add-a-custom-directory-to-the-image).
+
+You need to put your javaagent into your `src/main/jib` folder, so that this file will end in your containers file system.
```

Comment:
"You can add a javaagent to your container's file system by placing it in the `src/main/jib` folder then pointing to it with Jib's `container.jvmFlags` configuration." might be cleaner?

Reply:
This sentence seems to suggest that you drop it directly in `src/main/jib`, perhaps we should say "placing it *somewhere* in the `src/main/jib` directory" or something. Also **folder** is more of a UI construct, would prefer we use **directory** here.